plugins/pulp_rpm/plugins/migrations/0020_nested_drpm_directory.py

```diff
@@ -0,0 +1,119 @@
+import glob
+import logging
+import os
+import shutil
+import tempfile
+
+import lxml.etree as le
+
+from pulp.server.config import config
+from pulp.server.db.connection import get_collection
+
+
+_logger = logging.getLogger('pulp_rpm.plugins.migrations.0020')
+
+
+def _move_nested_drpm_dir(repo_dir):
+    """
+    If a repository has a nested <repo_dir>/drpm/drpm/ directory,
+    remove one of the nestings as it should not have been created
+    :param repo_dir: The root directory of the repository
+    :type repo_dir: str
+    """
+    drpm_dir = os.path.join(repo_dir, 'drpms', 'drpms')
+    target_drpm_dir = os.path.join(repo_dir, 'drpms')
+    if os.path.exists(drpm_dir):
```

Comment:
Should we worry about drpms/drpms/drpms? It seems like this will fix exactly one level of nesting.